frontend/src/Settings/Notifications/Notifications/EditNotificationModalContent.js

```diff
@@ -150,6 +154,32 @@ function EditNotificationModalContent(props) {
                 />
               </FormGroup>
 
+              <FormGroup>
+                <FormLabel>On Series Delete</FormLabel>
+
+                <FormInputGroup
+                  type={inputTypes.CHECK}
+                  name="onSeriesDelete"
+                  helpText="Be notified when series are deleted"
+                  isDisabled={!supportsOnSeriesDelete.value}
+                  {...onSeriesDelete}
+                  onChange={onInputChange}
+                />
+              </FormGroup>
+
+              <FormGroup>
+                <FormLabel>On Episode Delete</FormLabel>
```

Comment:
I think this should be `On Episode File Delete`?

Reply:
Done